app/(dashboard)/mailboxes/[mailbox_slug]/settings/integrations/connectSupportEmail.tsx

```
@@ -58,9 +44,25 @@ const ConnectSupportEmail = () => {
           </Link>
         </Alert>
       ) : (
-        <Button variant={supportAccount ? "destructive_outlined" : "subtle"} onClick={handleConnectOrDisconnect}>
-          {supportAccount ? `Disconnect ${supportAccount.email}` : "Connect your Gmail"}
-        </Button>
+        supportAccount ? ( <ConfirmationDialog
+          message="Are you sure you want to disconnect Gmail? You will still have access to all of your emails in Helper, but you will not be able to send/receive new emails until you connect a new Gmail account."
+          onConfirm={async ()=>{
```

ESLint

[error] Replace `=>` with `·=>·`

(prettier/prettier)